upstream/CVE-2019-11254.json

```
@@ -0,0 +1,60 @@
+{
+  "id": "CVE-2019-11254",
+  "modified": "2020-03-26T18:55:26Z",
+  "published": "2020-03-26T18:55:26Z",
+  "summary": "kube-apiserver Denial of Service vulnerability from malicious YAML payloads",
+  "details": "The Kubernetes API Server component in versions 0-1.14, and versions prior to 1.15.10, 1.16.8 and 1.17.3 allows an authorized user who sends malicious YAML payloads to cause the kube-apiserver to consume excessive CPU cycles while parsing YAML.",
```

Don't we need to fix the description?

```suggestion
  "details": "The Kubernetes API Server component in versions 0-1.14, and versions prior to 1.15.10, 1.16.7 and 1.17.3 allows an authorized user who sends malicious YAML payloads to cause the kube-apiserver to consume excessive CPU cycles while parsing YAML.",
```

---

Right! Thanks! Fixed in https://github.com/aquasecurity/vuln-list-k8s/pull/74/commits/aef5c9da9c29c2d11ec0e0b5436a82272f9e551c